.github/workflows/publish-docker-images.yml

```diff
@@ -83,9 +83,11 @@ jobs:
 
       - name: Setup Env
         run: |
-          echo "NAME=${{ matrix.name }}" >> $GITHUB_ENV
-          echo "TAG=${{ matrix.tag || 'latest' }}" >> $GITHUB_ENV
-          echo "PUSH=${{ matrix.push }}" >> $GITHUB_ENV
+          {
```

Comment:
fix linting `SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects`